typing/includemod.ml

```diff
@@ -731,7 +748,8 @@ and signature_components  ~in_eq ~loc old_env ~mark env subst
         match sigi1, sigi2 with
         | Sig_value(id1, valdecl1, _) ,Sig_value(_id2, valdecl2, _) ->
             let item =
-              value_descriptions ~loc env ~mark subst id1 valdecl1 valdecl2
+              core.value_descriptions ~loc env ~mark subst id1
```

Comment:
I'm not fond of the fact that, at this point, there *is* a `value_descriptions` function in scope, so that the code would work correctly if you forget to get the `core` version in scope. I think that it would be nicer if the `value_descriptions` function was renamed into `value_descriptions_inclusion` or `value_descriptions_subtyping` to avoid this.

Reply:
I have fixed this issue by grouping the inclusion function wrappers in their own `Core_inclusion` module.